test/PowerNotificationsTests/framework.h

```diff
@@ -0,0 +1,7 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#pragma once
+
+#define WIN32_LEAN_AND_MEAN             // Exclude rarely-used stuff from Windows headers
```

Comment:
Is there a reason to have this separate from pch.h? It's not a problem, explicitly. Just not sure what the benefit of splitting off this header is.